sf-fx-runtime-java-runtime/src/main/java/com/salesforce/functions/jvm/runtime/commands/AbstractDetectorCommandImpl.java

```diff
@@ -58,10 +59,35 @@ public final Integer call() throws Exception {
         projectPath,
         projectDetectDuration);
 
+    LOGGER.info("Reading project metadata at path {}...", projectPath);
+    ProjectMetadata projectMetadata =
+        ProjectMetadataParser.parse(projectPath.resolve("project.toml"))
+            .orElseThrow(
+                () -> new IllegalStateException("Cannot parse project metadata (project.toml)!"));
+
+    String salesforceApiVersion = Constants.DEFAULT_SALESFORCE_API_VERSION;
+
+    if (projectMetadata.getSalesforceApiVersion().isPresent()) {
+      salesforceApiVersion = projectMetadata.getSalesforceApiVersion().get();
+    } else {
+      LOGGER.warn(
+          "Project Salesforce API version isn't explicitly defined in project.toml. The default version {} will be used.",
```

Comment:
Should it be `Project's Salesforce API version` to indicate its ownership(?) Or we can remove the word Project altogether maybe?

Reply:
Fixed in c67a1a8, thanks! :)